### DISCUSSION TITLE
logout shield

### DISCUSSION THREAD

**Discussion by helmihasibuan** [codeigniter4/shield#551] [Q&A] [answered]
hi,
i want to ask how to logout from webpage with shield function?
i add **/Logout** on logout button it cant work, but if i type manually **http://localhost:8080/logout** it work.

thank you

**Comment by kenjis** [comment, marked as answer]
What do you mean "add /Logout on logout button"?

The URI path is `/logout` not `/Logout`, and you need to send a GET request to it.
